flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlSource.java

```diff
@@ -200,7 +201,22 @@ public SplitEnumerator<MySqlSplit, PendingSplitsState> createEnumerator(
         validator.validate();
 
         final MySqlSplitAssigner splitAssigner;
-        if (!sourceConfig.getStartupOptions().isStreamOnly()) {
+        // In batch mode, only split snapshots.
```

Comment:
This change will also modify the behavior of Stream mode, should we update the comment to `In snapshot-moly mode`?